network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt

```diff
@@ -46,8 +49,11 @@ enum class NetworkProtectionPixelNames(
     NETP_WG_ERROR_CANT_START_WG_BACKEND_DAILY("m_netp_ev_wireguard_error_cannot_start_wireguard_backend_d", enqueue = true),
     NETP_WG_ERROR_FAILED_TO_LOAD_WG_LIBRARY("m_netp_ev_wireguard_error_unable_to_load_wireguard_library_c", enqueue = true),
     NETP_WG_ERROR_FAILED_TO_LOAD_WG_LIBRARY_DAILY("m_netp_ev_wireguard_error_unable_to_load_wireguard_library_d", enqueue = true),
-    NETP_LATENCY_REPORT("m_netp_ev_latency_c"),
-    NETP_REPORT_POOR_LATENCY_DAILY("m_netp_ev_poor_latency_d"),
+    NETP_REPORT_TERRIBLE_LATENCY("m_netp_ev_terrible_latency_c"),
```

Comment:
Should terrible and poor be ennqueued? Not sure about the likelihood of failing to send this pixel if the device connectivity is bad?

Reply:
I think it's fine not to enqueue them (we can revisit once we see the first metrics). The reasoning is:
* if we're able to measure latency pixels should go through
* even if they don't we compute a running geometric average, and the weight on the last value is such that the average is not extremely responsive to it, so the poor connectivity will take several good measurements to go back to be GOOD